### DISCUSSION TITLE
How to make import maps work in a monorepo setting?

### DISCUSSION THREAD

**Discussion by devlzcode** [denoland/deno#17797] [Q&A]
Its actually impossible to have a working monorepo with multiple Deno folders without having some error in regards to the LSP or linter unless the intended usuage of a workspace is to have multiple windows open instead of the one? I don't know.

**Comment by bradenmacdonald** [comment]
I use a monorepo with VS Code's "multi-root workspaces" feature and it works fine. Some of the folders have import maps, others don't, and one is even a node app, but they all work properly. Can you explain the question more?
